CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ This release drops the compatibility guarantee of [Go 1.18].
 
 - Handle empty environment variable as it they were not set. (#3764)
 - Clarify the `httpconv` and `netconv` packages in `go.opentelemetry.io/otel/semconv/*` provide tracing semantic conventions. (#3823)
+- TracerProvider consistently doesn't allow to register a SpanProcessor after shutdown (#3845)
```

Comment:
```suggestion
- TracerProvider consistently doesn't allow to register a SpanProcessor after shutdown. (#3845)
```

(I know this is a huge nit, it's more for consistency)

Reply:
Done. Actually, the line was in a wrong section so moved it.